### PR TITLE
jetty 9.3.7.v20160115

### PR DESCRIPTION
**Pull request by niki-timofe**
Fixed broken link

### CODE REVIEW

**Comment by apjanke** [comment]
This PR reformatted the formula to a 4-space indent. Brew uses a 2-space indent, and we prefer to minimize formatting changes in version bumps, to make diffs and commit history readable. Could you revert the formatting changes?

**Comment by niki-timofe** [comment]
Done

File: Library/Formula/jetty.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ def install
       (bin+scriptname).write <<-EOS.undent
         #!/bin/bash
         JETTY_HOME=#{libexec}
-        #{f} "$@"
+      #{f} "$@"
```
Comment (dunn) [inline Library/Formula/jetty.rb]: Was this intentional?

**Comment by DomT4** [comment]
Any reason for the move from `.tar.gz` to `.zip`, out of interest?

**Comment by niki-timofe** [comment]
Just because I usually use `.zip`

**Comment by DomT4** [comment]
If the `tar.gz` link has the same checksum it might be nice to keep using that one so people don't see an unexplained checksum change in the `diff`?

**Comment by makigumo** [comment]
Just FYI, the old link is still working and can be found on the official download page (http://download.eclipse.org/jetty/). The proposed ```stable-9/dist``` link will break, when a new version is released, because it only contains the latest version (http://download.eclipse.org/jetty/stable-9/dist/).
So I don't think that this PR is necessary.

**Comment by dunn** [comment]
The old URL is working for me too, so I'll go ahead and close this.  But thank you for the submission!

**Pull request closed without merging**